### DISCUSSION TITLE
Is there a way to sharpen an image with Magick.NET?

### DISCUSSION THREAD

**Discussion by alvipeo** [dlemstra/Magick.NET#1706] [Help]
I've successfully tried image resizing, but I also need to sharpen an image. Is there any way to do this in C#? I didn't find it anywhere.
Thanks!

**Comment by dlemstra** [comment]
Have you tried to use `image.Sharpen(`?

**Reply by alvipeo** [reply]
Ooooopsy..) no. But it's not mentioned anywhere...

**Reply by dlemstra** [reply]
No worries. Visual Studio should help you with intelisense to find some of the available features.

**Comment by alvipeo** [comment]
ok, thanks!
